### PR TITLE
Fixes deconz otau download issues

### LINKED ISSUE #607: deconz ota download scripts are broken
For the ota tools the following tools are missing:
- iproute2
- wget
- iputils-ping

further the directory /data/otau/ isn't created before downloading, resulting in more errors

### PR DESCRIPTION
**Pull request by mjrider**
Fixes #607 the problem that ota files are not downloaded

### CODE REVIEW

File: deconz/data/ika-otau-dl.sh [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 URL_IKEA="http://fw.ota.homesmart.ikea.net/feed/version_info.json"
 
+mkdir "/data/otau"
```
Comment (pvizeli) [inline deconz/data/ika-otau-dl.sh]: Create this on `run.sh` before we start OTA scripts. `mkdir -p ....`

File: deconz/Dockerfile [hunk 2]
```diff
@@ -22,6 +22,9 @@ RUN apt-get update \
         libqt5websockets5 \
         libqt5widgets5 \
         sqlite3 \
+	iputils-ping \
+	iproute2 \
```
Comment (pvizeli) [inline deconz/Dockerfile]: why iproute2 ?

Comment (mjrider) [reply deconz/Dockerfile]: because the command `ip` is in that package, and deconz is using that for the detecting of working internet

**Comment by pvizeli** [comment]
Update version +0.1 and add a release node

**Comment by mjrider** [comment]
@pvizeli do you agree with merging this ?

**Pull request merged**